web_widget_dropdown_dynamic/static/tests/web_widget_dropdown_dynamic_tests.esm.js

```diff
@@ -1,8 +1,6 @@
-/** @odoo-module **/
-
+/* eslint-disable no-undef */
```

Comment:
Why disabling this linter?

Reply:
Because it's giving the below error

```
web/web_widget_dropdown_dynamic/static/tests/web_widget_dropdown_dynamic_tests.esm.js
    7:1  error  'QUnit' is not defined  no-undef
   38:5  error  'QUnit' is not defined  no-undef
   91:5  error  'QUnit' is not defined  no-undef
  122:5  error  'QUnit' is not defined  no-undef
  149:5  error  'QUnit' is not defined  no-undef

✖ 5 problems (5 errors, 0 warnings)
```

Reply:
Yeah, and that's because you are removing the import, so for me, you shouldn't do it. I don't see the relation of this with the error in #3136 @CarlosRoca13 can you maybe help here?

Reply:
In odoo 18.0, is not needed to import QUnit because it is declared as global https://github.com/odoo/odoo/blob/4b6770ec30bd00213674df2b075c77f33a0e65d9/addons/web/static/src/%40types/libs.d.ts#L5

So we should add here the key https://github.com/OCA/web/blob/514c425611706f67bf6d4957e7942ef4da3826a2/eslint.config.cjs#L10-L21

Reply:
Shouldn't that be in the copier template?

Reply:
Yes, sure

Reply:
:pray:

Reply:
https://github.com/OCA/oca-addons-repo-template/pull/301

Reply:
Done, thanks